src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * Math.pow(radius, 2);
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: Circle, " + "area: " + getArea() + " sq.units, "
+                + "radius: " + radius + " units, " + "color: " + getColor();
+    }
+
+    @Override
+    public String draw() {
+        return toString();
+    }
```

Comment:
Do not override toString and change draw()

![image](https://user-images.githubusercontent.com/20326858/210266143-baf4da30-02c1-4da3-b5c4-c68f58a7f414.png)